bedrock/firefox/templates/firefox/onboarding/user-actions-mobileprivacy.html

```diff
@@ -0,0 +1,51 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/onboarding/onboarding-base.html" %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_onboarding_user_actions' %}
+{% endblock %}
+
+{% block title %}
+  {{ _('To the Internet and beyond') }}
+{% endblock %}
+
+{% block onboarding_content %}
+  <ul id="user-actions">
+    <li class="user-action mobile">
+      <h3>{{ _('Take it everywhere you go') }}</h3>
+
+      <p>
+        {{ _('Get the speed and safety of Firefox on your smartphone and tablet.') }}
+      </p>
+
+      <ul id="mobile-dl-buttons">
+        <li>
+          <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" data-link-type="download" data-download-os="Android">
+            {{ high_res_img('firefox/android/btn-google-play.png', {'alt': _('Get it on Google Play'), 'width': '152', 'height': '45', 'l10n': True}) }}
+          </a>
+        </li>
+        <li>
+          <a rel="external" href="{{ firefox_ios_url('mozorg-fx-onboarding-mobileprivacy') }}" data-link-type="download" data-download-os="iOS">
+            <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+          </a>
+        </li>
+      </ul>
+    </li>
+    <li class="user-action privacy">
+      <h3>{{ _('Browse like no one’s watching') }}</h3>
+
+      <p>
+        {{ _('Use Private Browsing with Tracking Protection to cover your tracks.') }}
+      </p>
+
+      <a href="https://support.mozilla.org/kb/private-browsing-use-firefox-without-history/" class="trailing-arrow" id="try-pb" data-highlight="privateWindow">{{ _('Try it now') }}</a>
```

Comment:
Could a better destination for this link be the `/firefox/private-browsing` page?